concepts/05 Widgets/Toast/05 Show and Hide the Toast/10 Events.md

```diff
@@ -1,10 +1,10 @@
 To execute certain commands before or after the **Toast** was shown/hidden, handle the [showing](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/showing.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#showing'), [shown](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/shown.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#shown'), [hiding](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/hiding.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#hiding') or [hidden](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/hidden.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#hidden') event. If the event handling function is not going to be changed during the lifetime of the widget, assign it to the corresponding **on*EventName*** option. For example, in the following code, a handler of the **hidden** event is assigned to the [onHidden](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/onHidden.md '/Documentation/ApiReference/UI_Widgets/dxToast/Configuration/#onHidden') option. This handler counts down from three replacing the message in the **Toast** at the same time.
 
 ---
-#####jQuery
+##### jQuery
```

Comment:
This code sample is really strange. Let's use a generic example instead. Something like:

```
onHiding: () => {
    // Handler of the "hiding" event
},
onHidden: () => {
    // Handler of the "hidden" event
},
```

and so on.

Reply:
fixed